src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/StatementOfMeansContentProvider.java

```diff
@@ -0,0 +1,199 @@
+            .put("amountPaid", formatMoney(expense.getAmountPaid()))
+            .put("frequency", expense.getFrequency().getDescription())
+            .build();
+    }
+
+    private Map<String, Object> createEmployment(StatementOfMeans statementOfMeans) {
+        ImmutableMap.Builder<String, Object> contentBuilder = ImmutableMap.builder();
+
+        statementOfMeans.getEmployment().ifPresent(employment -> {
+            contentBuilder.put("jobType", createJobType(employment));
+            contentBuilder.put("employment", employment);
+            employment.getSelfEmployment().ifPresent(selfEmployment -> {
+                contentBuilder.put("selfEmployment", createSelfEmployment(selfEmployment));
+                selfEmployment.getOnTaxPayments().ifPresent(onTaxPayments ->
+                    contentBuilder.put("onTaxPayments", createOnTaxPayments(onTaxPayments))
+                );
+            });
+        });
+
+        return contentBuilder.build();
+    }
+
+    private Map<String, Object> createSelfEmployment(SelfEmployment selfEmployment) {
+        return new ImmutableMap.Builder<String, Object>()
+            .put("jobTitle", selfEmployment.getJobTitle())
+            .put("annualTurnover", formatMoney(selfEmployment.getAnnualTurnover()))
+            .build();
+    }
+
+    private Map<String, Object> createOnTaxPayments(OnTaxPayments onTaxPayments) {
+        return new ImmutableMap.Builder<String, Object>()
+            .put("amountYouOwe", formatMoney(onTaxPayments.getAmountYouOwe()))
+            .put("reason", onTaxPayments.getReason())
+            .build();
+    }
+
+    private Map<String, Object> createDependant(StatementOfMeans statementOfMeans) {
+        ImmutableMap.Builder<String, Object> contentBuilder = ImmutableMap.builder();
+
+        statementOfMeans.getDependant().ifPresent(dependant -> {
+            contentBuilder.put("dependant", dependant);
+            contentBuilder.put("children", dependant.getChildren()
+                .stream()
+                .map(this::createChild)
+                .collect(toList())
+            );
+            dependant.getOtherDependants().ifPresent(otherDependants -> {
+                contentBuilder.put("otherDependants", otherDependants);
+            });
+            dependant.getNumberOfMaintainedChildren().ifPresent(
+                maintainedChildren -> contentBuilder.put("maintainedChildren", maintainedChildren)
+            );
+        });
+
+        return contentBuilder.build();
+    }
+
+    private Map<String, Object> createChild(Child child) {
+        requireNonNull(child);
+
+        return new ImmutableMap.Builder<String, Object>()
+            .put("ageGroupType", child.getAgeGroupType().getDescription())
+            .put("numberOfChildren", child.getNumberOfChildren())
+            .put("numberOfChildrenLivingWithYou", child.getNumberOfChildrenLivingWithYou())
+            .build();
+    }
+
+    public String createJobType(Employment employment) {
```

Comment:
We should not OPEN API for testing purpose only. That encourages using that method for other purposes and increases change of unwanted / unexpected coupling.

Either tests is via main public method or extract to inner class like `StatementOfMeansContentProvider.EmploymentContentProvider`